apollo-router/src/configuration/upgrade.rs

```diff
@@ -144,7 +144,7 @@ fn apply_migration(config: &Value, migration: &Migration) -> Result<Value, Confi
                 }
             }
             Action::Change { path, from, to } => {
-                if !jsonpath_lib::select(config, &format!("$.{path} == {from}"))
```

Comment:
FWIW this syntax just seemed to be wrong (and the library ignored everything after the space without an error?)

https://freestrings.github.io/jsonpath/ can be used to test this jsonpath_lib implementation.